src/parcsr_ls/par_amg_setup.c

```diff
@@ -750,6 +767,7 @@ hypre_BoomerAMGSetup( void               *amg_vdata,
    Vtemp = hypre_ParVectorCreate(hypre_ParCSRMatrixComm(A_array[0]),
                                  hypre_ParCSRMatrixGlobalNumRows(A_array[0]),
                                  hypre_ParCSRMatrixRowStarts(A_array[0]));
+   hypre_ParVectorNumVectors(Vtemp) = num_vectors;
```

Comment:
How about `hypre_ParMultiVectorCreate`?

Reply:
That would work also, but my understanding is that this routine does not align well with our vector framework, i.e., the `hypre_ParVector` functions should start with `hypre_ParVector` while this one has `hypre_ParMultiVector`. So, I decided to not use it and eventually it could be removed. @rfalgout, do you agree with this reasoning?

Reply:
Yes.  I think we decided to just add a `SetNumComponents` call to the standard `Vector` interface to keep backward compatibility for users.  So, a `Vector` in hypre *is* a multi-vector, and there is no need for two concepts.  I think this conforms to that thinking, if I understand the question correctly.

Reply:
Yes, I was planning to add the `SetNumComponents` call in a separate PR since I'll need to update several files in hypre and wanted to make the review of this PR easier. So, in the future, `hypre_ParVectorNumVectors(Vtemp) = num_vectors` will be `hypre_ParVectorSetNumComponents(Vtemp, num_vectors)`. Does that sound good?

Reply:
Yep!

Reply:
Then, is there any reason to keep `hypre_ParMultiVectorCreate`? I see @rfalgout mentioned the "backward compatibility" issue. Just asking.